### DISCUSSION TITLE
Configurable interactive resize modifier

### DISCUSSION THREAD

**Discussion by notpeelz** [YaLTeR/niri#682] [Ideas]
Would it be possible to make the mouse resize modifier configurable?
I have my caps lock mapped to Mod5 via my xkb config, and I use mod5 for all my shortcuts. It would feel more natural if I could also bind the mouse resize modifier to Mod5.

**Comment by YaLTeR** [comment]
It's not configurable at the moment, it's equal to Mod. But it shouldn't be hard to edit in the code, in input.rs mouse button event
